src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery ballSupplier = new Lottery();
+        Ball[] arrayOfThreeBalls = new Ball[3];
+        for (int i = 0; i < arrayOfThreeBalls.length; i++) {
+            arrayOfThreeBalls[i] = ballSupplier.getRandomBall();
+        }
+        for (Ball ball : arrayOfThreeBalls) {
+            System.out.println(ball);
+        }
```

Comment:
make only one loop, where you will get one ball per loop iteration, and print info about that. without any arrays